src/services/postService.js

```diff
@@ -99,5 +35,62 @@ module.exports = {
       where: {children: {[Op.gte]: 0} }
     })
       .catch(err => console.log(err));
-  }
+  },
+  determinePostType: function(links) {
+     if (!links) {
+       // blog
+       return 0;
+     } else if (this.containsVideo(links)) {
+       // is video
+       return 1;
+     } else {
+       // news
+       return 2;
+     }
+  },
```

Comment:
We should return more dev-friendly values here.
something like
```
const BLOG_POST = 0;
const VIDEO_POST = 0;
const NEWS_POST = 0;

determinePostType: function(links) {
   if (!links) {
      // blog
      return BLOG_POST;
   } else if (this.containsVideo(links)) {
      // is video
      return VIDEO_POST;
   } else {
      ...
   }
}
```

Reply:
or even

```
const BLOG_POST = 'blog_post';
const VIDEO_POST = 'video_post';
const NEWS_POST = 'news_post';
```

This way we're working with descriptive, easy to understand variable names instead of obscure numbers